apps/ios/SimpleX Localizations/en.xcloc/Localized Contents/en.xliff

```diff
@@ -587,6 +592,11 @@ to scan from the app</target>
         <target>Unexpected error: %@</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Unless your contact deleted the connection or this link was already used, it might be a bug - please report it. To connect, please ask your contact to create another connection link and check if you have a stable network connection." xml:space="preserve">
+        <source>Unless your contact deleted the connection or this link was already used, it might be a bug - please report it. To connect, please ask your contact to create another connection link and check if you have a stable network connection.</source>
+        <target>Unless your contact deleted the connection or this link was already used, it might be a bug - please report it. To connect, please ask your contact to create another connection link and check if you have a stable network connection.</target>
```

Comment:
```suggestion
        <target>Unless your contact deleted the connection or this link was already used, it might be a bug - please report it.\nTo connect, please ask your contact to create another connection link and check if you have a stable network connection.</target>
```